### PR TITLE
Bug 1220716 - Simplify storage of talos data in perfherder

### PR DESCRIPTION
**Pull request by wlach**
* Stop storing mean/median/max/min for subtests as they're not always on the
  same scale as the summary value
* Always use "filtered" value provided by Talos for subtest summary values
* Store value for tests as a "value" property for the series data

[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/934)

### CODE REVIEW

File: ui/partials/perf/graphsctrl.html [hunk 1]
```diff
@@ -60,8 +54,7 @@
       <div>
         <p id="tt-series"><span ng-bind="tooltipContent.test"/>
           (<span ng-bind="tooltipContent.project.name"/>)</p>
-        <p id="tt-series2" class="small"><span ng-bind="tooltipContent.platform"/>
-          (<span ng-bind="tooltipContent.machine"/>)</p>
+        <p id="tt-series2" class="small"><span ng-bind="tooltipContent.platform"/></p>
```
Comment (jmaher) [inline ui/partials/perf/graphsctrl.html]: why is this removed?

**Comment by jmaher** [comment]
do we need to update the unittests as well?

File: ui/partials/perf/graphsctrl.html (hunk 1 above)
Comment (wlach) [inline ui/partials/perf/graphsctrl.html]: machine was never defined, so this always wound up beig displayed as (mean), which is misleading. I opted to just take it out (technically I suppose I could/should file a seperate bug for this)

**Comment by wlach** [comment]
Yeah we definitely do need to update the tests. I'll take care of this.

File: tests/etl/test_perf_data_adapters.py [hunk 2]
```diff
@@ -63,16 +63,14 @@ def test_adapt_and_load(self):
...
                     # it in
-                    for measure in ['min', 'max', 'std', 'mean', 'median']:
-                        self.assertEqual(
-                            round(talos_datum['summary']['subtests'][testname][measure], 2),
-                            perfdata[measure])
+                    self.assertEqual(
+                        round(talos_datum['summary']['subtests'][testname]['filtered'], 2),
+                        perfdata['value'])
                 else:
                     # this is an old style talos blob without a summary. these are going
```
Comment (avih) [inline tests/etl/test_perf_data_adapters.py]: Here and on other places which ingest the "legacy" `value` field, do we expect TH to ever bump into a talos logs without a `filtered` field? I.e. right now AFAIK all the logs have `filtered`, so unless there's a good reason to keep them (like maybe the talos logs for the beta branch etc don't have `filtered`), I'd say just remove all these legacy "else" clauses.

File: treeherder/etl/perf_data_adapters.py [hunk 3]
```diff
@@ -101,8 +95,7 @@ def _calculate_test_data(job_id, result_set_id, push_timestamp,
             "job_id": job_id,
             "result_set_id": result_set_id,
             "push_timestamp": push_timestamp,
-            "total_replicates": r_len,
```
Comment (avih) [inline treeherder/etl/perf_data_adapters.py]: Do we not want to store the number of replicates the value was generated from? it ranges from 1 (v8_7) to dozens (dromaeo), and anything in between.

File: tests/etl/test_perf_data_adapters.py (hunk 2 above)
Comment (wlach) [inline tests/etl/test_perf_data_adapters.py]: Unfortunately the Android Talos tests don't have this summary information, so we still need to process this type of information from them as long as they're turned on. Hopefully won't be that much longer. After that, I'll be happy to clean this up. :)

File: treeherder/etl/perf_data_adapters.py (hunk 3 above)
Comment (wlach) [inline treeherder/etl/perf_data_adapters.py]: We're not using it for anything right now, and leaving it out will let us simplify some things in the larger scale performance refactoring I'm doing. It will be easy to re-add it in the future if there is justification.

**Comment by wlach** [comment]
This was merged in 0b79d549d6
